.cirrus.yml

```diff
@@ -89,28 +89,42 @@ on_failure_googlecompute: &on_failure_googlecompute
 
 
 task:
+  name: 'Building VM Image: ${IMAGE_NAME}'
+  alias: ${IMAGE_NAME}
   matrix:
-    - name: freebsd-13
-      env:
+    - env:
+        IMAGE_NAME: freebsd-13
         PACKERFILE: packer/freebsd.pkr.hcl
 
     - env:
+        IMAGE_NAME: bullseye
+        PACKERFILE: packer/linux_debian.pkr.hcl
+        SCRIPTS: scripts/linux_debian_*
+
+    - env:
+        IMAGE_NAME: sid
+        PACKERFILE: packer/linux_debian.pkr.hcl
+        SCRIPTS: scripts/linux_debian_*
+
+    - env:
+        IMAGE_NAME: sid-newkernel
         PACKERFILE: packer/linux_debian.pkr.hcl
         SCRIPTS: scripts/linux_debian_*
 
-      matrix:
-        - name: bullseye
-        - name: sid
-        - name: sid-newkernel
-        - name: sid-newkernel-uring
+    - env:
+        IMAGE_NAME: sid-newkernel-uring
+        PACKERFILE: packer/linux_debian.pkr.hcl
+        SCRIPTS: scripts/linux_debian_*
```

Comment:
Hm, what's the advantage of getting rid of this matrix?

Reply:
Fixed.